### ISSUE TITLE
Different data results from PR comment to app

### ISSUE THREAD

**Issue by codecovdesign** [codecov/engineering-team#1321]
**Describe the bug**
Report of different data shown on PR comment and app, see here: https://github.com/codecov/feedback/issues/8#issuecomment-1970889554

-  patch reporting as 92.80303% in PR report and 95.83% in-app
- missing lines is 19 in pr report and 5 in-app (changed lines)

![Screenshot 2024-02-29 at 8 29 01 AM](https://github.com/codecov/engineering-team/assets/87861036/12d2486e-37f5-4d99-bcad-bf483a83ef4e)
![Screenshot 2024-02-29 at 8 29 08 AM](https://github.com/codecov/engineering-team/assets/87861036/b8f93fec-8c5b-4377-ad88-435e56b405ea)

Related PR

**Expected behavior**
Consistent data display of project, patch, and lines missin

**Comment by giovanni-guidini** [comment]
Looks like the numbers stabilized afterwards...

![Image](https://github.com/codecov/engineering-team/assets/99758426/235b17b9-31eb-43ab-9e0c-0c9975f735e7)



![Image](https://github.com/codecov/engineering-team/assets/99758426/ca117b86-7c75-4acd-8bec-3bad13947970)

The customer in question also mentioned this:

> The above library that I linked is sending multiple coverage files since some of the tests run on different workers in parallel.
Codecov will create a coverage comment when the first file al finishes processing, so I get a mail notifying me with wrong coverage. In most cases this coverage comment will be updated although I've seen cases where it didn't.
I would consider posting the coverage comment only when all the runners complete.

It is very likely that we simply failed to send one of the notifications so the PR comment didn't update to the value displayed in the UI. As more commits were added to the PR new notifications were issues and both reports were brought to parity.
